BLSQ/iaso
# IA-2546: Data Source read-write permissions

Explain what problem this PR is resolving
- The Geo data sources - Read and Write was missing in the permissions list
- This PR is for adding the missing permission
Related JIRA tickets : [IA-2546](https://bluesquare.atlassian.net/browse/IA-2546)

## Self proofreading checklist


## Changes
- Adding the **Geo data sources - Read and Write** permission in the MODULE_PERMISSIONS
- Add the permission in the datasourcePath

## How to test
- Edit or create a user
- Check if the permission is in the list

## Print screen / video
![Screenshot from 2023-12-05 15-40-18](https://github.com/BLSQ/iaso/assets/19631540/917764a5-def5-4b77-b24e-8c54fcaba876)


[IA-2546]: https://bluesquare.atlassian.net/browse/IA-2546?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
